JM/CF/Scripts/1_Core/CommunityFramework/Logging/CF_Trace.c

```diff
@@ -202,310 +202,372 @@ class CF_Trace : Managed
 
 static CF_Trace CF_Trace_Instance(Class instance)
 {
-	return new CF_Trace(CF_Trace.FormatInstance(instance), string.Empty, 0, true);
+	auto trace = new CF_Trace(CF_Trace.FormatInstance(instance), string.Empty, 0, true);
+	return trace;
```

Comment:
What is the idea behind this?

Reply:
Assigning to a variable so it can have it's ref count set.